dependencies.props

```diff
@@ -28,6 +29,7 @@
     <ProjectNTfsTestILCPackageVersion>1.0.0-beta-25308-00</ProjectNTfsTestILCPackageVersion>
     <NETStandardPackageVersion>2.0.0-preview2-25308-01</NETStandardPackageVersion>
     <NETStandardPackageId>NETStandard.Library</NETStandardPackageId>
+    <CoreSetupPackageVersion>2.0.0-preview2-002093-00</CoreSetupPackageVersion>
```

Comment:
I slightly prefer giving these properties explicit names, like `MicrosoftNETCoreAppPackageVersion` in this case. Precise and avoids overlap if we ever depend on another Core-Setup package in the future with a different version. Other props here don't follow the convention, up to you.